UT4MasterServer/Helpers/StringExtensions.cs

```diff
@@ -19,4 +19,15 @@ public static bool IsHexString(this string input)
 
 		return true;
 	}
+
+	public static bool IsBase64(this string input)
+	{
+		if (string.IsNullOrWhiteSpace(input))
+		{
+			return false;
+		}
+
+		var buffer = new Span<byte>(new byte[input.Length]);
+		return Convert.TryFromBase64String(input, buffer, out _);
```

Comment:
Instead of just checking `IsBase64` we could **additionally** expose `string.TryDecodeBase64` with an `output` parameter for decoded string.

Reply:
Done